en/components/tree.md

```diff
@@ -0,0 +1,259 @@
+- [**expand**]({environment:angularApiUrl}/classes/igxtreenodecomponent.html#expand) - expands the node
+- [**collapse**]({environment:angularApiUrl}/classes/igxtreenodecomponent.html#collapse) - collapses the node
+- [**toggle**]({environment:angularApiUrl}/classes/igxtreenodecomponent.html#toggle) - toggles node expansion state
+- [**collapseAll**]({environment:angularApiUrl}/classes/igxtreecomponent.html#collapseAll) - collapses the specified nodes. If no nodes are passed, collapses all parent nodes.
+- [**expandAll**]({environment:angularApiUrl}/classes/igxtreecomponent.html#expandAll) - sets the specified nodes as expanded. If no nodes are passed, expands all parent nodes.
+- [**deselectAll**]({environment:angularApiUrl}/classes/igxtreecomponent.html#deselectAll) - deselects all nodes. If a nodes array is passed, deselects only the specified nodes. Does not emit nodeSelection event.
+
+### Finding Nodes
+You can find a specific node within IgxTree by using the [findNodes]({environment:angularApiUrl}/classes/igxtreecomponent.html#findNodes) method.  It returns an array of nodes, which match the specified data.
+When finding nodes, you can pass a custom comparer function in order to find the data.
+```html
+<igx-tree>
+	<igx-tree-node *ngFor="let nodes of data" [data]="node" [expanded]="isNodeExpaded(node)" [selected]="isNodeSelected(node)">
+		{{ node.text }}
+		<img [src]="node.image" alt="node.imageAlt" />
+		<igx-tree-node *ngFor="let child of node.children" [data]="child" [expanded]="isNodeExpaded(child)" [selected]="isNodeSelected(child)">
+      {{ child.text }}
+		</igx-tree-node>
+	</igx-tree-node>
+</igx-tree>
+```
+```typescript
+export class MyTreeViewComponent {
+  public data: { [key: string]: any, valueKey: string } = MY_DATA;
+  @ViewChild(IgxTreeComponent, { read: IgxTreeComponent })
+  public tree;
+
+  findNode(valueKey: string) {
+    const comparer: (nodeData: any, node: IgxTreeNodeComponent) => boolean =
+      (data: any, node: IgxTreeNodeComponent) => nodeData.valueKey === data;
+    this.tree.findData(valueKey, comparer);
+  }
+}
+```
+### Templating
+To create a custom node template within igx-tree, declare `<ng-template>`. From the template you can access the implicitly provided context variable, with properties that give you information about the node. 
+```html
+ <igx-tree>
+    <igx-tree-node #node>
+        <ng-template *ngTemplateOutlet="nodeTemplate; context { $implicit: node }">
+        </ng-template>
+    </igx-tree-node>
+    <ng-template #nodeTemplate let-node>
+        <igx-combo type="line" [(ngModel)]="node.CompanyName" [displayKey]="'CompanyName'" [data]="data" width="220px"></igx-combo>
+    </ng-template>
+</igx-tree>
+```
+Additionally, by using the [expandIndicator]({environment:angularApiUrl}/classes/igxtreecomponent.html#expandIndicator) input you have the ability to set a custom template to be used for rendering the expand/collapse indicators of nodes.
```

Comment:
It's actually advisable (since we're going for a declarative-first approach) to pass the template w/ `[igxTreeExpandIndicator]` directive:
```html
<igx-tree>
    <igx-tree-node *ngFor="let node of data" [data]="node">
    </igx-tree-node>
    <ng-template igxTreeExpandIndicator let-expanded>
        <igx-icon>{{ expanded ? 'expand_less' : 'expand_more' }}</igx-icon>
    </ng-template>
</igx-tree>
```